projects/components/src/form/form-select/form-select.component.spec.ts

```diff
@@ -17,6 +17,11 @@ export class VcdFormSelectWidgetObject extends WidgetObject<FormSelectComponent>
         return this.findElement('select').nativeElement;
     }
 
+    get clrIcon(): string {
```

Comment:
Please rename to `clrIconShape`. Add some documentation explaining that it returns an empty string  if there's no icon

Reply:
done